airflow/models/dagrun.py

```diff
@@ -1248,7 +1259,7 @@ def get_run(session: Session, dag_id: str, execution_date: datetime) -> DagRun |
             session.query(DagRun)
             .filter(
                 DagRun.dag_id == dag_id,
-                DagRun.external_trigger == False,  # noqa
+                DagRun.external_trigger is False,
```

Comment:
Interesting that `is` works

Reply:
this was not intentional... weird... i'll revert this line

Reply:
maybe a pre-commit was added?